src/mail.sh

```diff
@@ -31,6 +31,11 @@ function mail_main()
     return 22 # EINVAL
   fi
 
+  if [[ "${options_values['SEND']}" == 1 ]]; then
+    mail_send ''
```

Comment:
Shellcheck fails without the argument. This is because this function takes the `flag` as an argument and it is only called here. [SC2120](https://github.com/koalaman/shellcheck/wiki/SC2120)

Reply:
> If the parameters are optional and you currently just don't want to use them, you can ignore this message. In versions strictly greater than v0.6.0, ignoring SC2120 on a function will also disable SC2119 on each of the call sites.

I think we are in the exception case here. Maybe you can just ignore this shellcheck here.